emData/generate_PR.py

```diff
@@ -0,0 +1,103 @@
+# Automatically generates ProjectionRouterTop.h, and ProjectionRouterTop.cc
+import sys, os
+
+modules_to_build = ["PR_L1PHIB","PR_L2PHIB","PR_L3PHIB","PR_L4PHIB","PR_L5PHIB","PR_L6PHIB",
+                    "PR_L1PHIC","PR_L2PHIC","PR_L3PHIC","PR_L4PHIC","PR_L5PHIC","PR_L6PHIC"]
```

Comment:
@djcranshaw Is there any reason we can't expand this list to include all PRs, or at least all barrel PRs? We don't need to test them all right now, but it would be nice if they were all defined.